examples/3d/volumetric_fog.rs

```diff
@@ -60,6 +61,44 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
             ..default()
         });
 
+    // Add the point light
+    commands
+        .spawn(PointLightBundle {
+            point_light: PointLight {
+                shadows_enabled: true,
+                range: 50.0,
+                color: RED.into(),
+                intensity: 1000.0,
+                ..default()
+            },
+            transform: Transform::from_xyz(-0.3493744, 1.900556, 1.0452124),
```

Comment:
Nit: can we pick some slightly rounder numbers to make reading the example easier?